tests/swift-sdk-swift-tests/AuthTests.swift

```diff
@@ -573,7 +573,7 @@ class AuthTests: XCTestCase {
         condition1.expectedFulfillmentCount = 2
         
         let authDelegate = createAuthDelegate({ completion in
-            completion(nil)
+            completion(AuthTests.authToken)
```

Comment:
I think we should check for expectations here? Otherwise `authDelegate` in this test is basically not being tested at all. We might as well not set it.